src/UCTNode.cpp

```diff
@@ -322,15 +362,25 @@ UCTNode* UCTNode::uct_select_child(int color) {
         }
     }
 
+    if (parentvisits == 0) {
+        // sort by score.
+        parentvisits = 1;
```

Comment:
I believe this is a no-op.

Before children had been sorted by score then would have equal eval and first child (with best score) would have been chosen.